adafruit_guvx_i2c.py

```diff
@@ -28,11 +28,18 @@
 """
 
 import time
+from tkinter.messagebox import NO  # pylint: disable=unused-import
```

Comment:
I think this was accidentally commited, it definitely isn't needed :smile:

Reply:
I will remove that comment. I added it because of a pylint error.